### PR TITLE
Various improvements and clarifications

### PR DESCRIPTION
**Pull request by cristian-recoseanu**
- protocolVersion field removed from IS-12
- add specific section for "Command response message type" classId naming:
- GetControlClass (identity -> classId)
- NcClassDescriptor (identity -> classId)
- correct JSON examples

### CODE REVIEW

File: docs/Protocol messaging.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # Protocol messaging
 
-All protocol messages MUST have have the following 2 properties:
+All protocol messages MUST have have the following property:
```
Comment (jonathan-r-thorpe) [inline docs/Protocol messaging.md]: Given this is now a list of one, could this be collapsed into a single sentence?
```suggestion
All protocol messages MUST have the messageType property to indicate the type of the message (e.g. Command).
```

File: docs/Protocol messaging.md [hunk 2]
```diff
@@ -1,8 +1,7 @@
...
 
-* protocolVersion - describes the version of this protocol (e.g. 1.0.0)
 * messageType - describes the message type (e.g. Command)
 
 `Note`: When sending messages, each message MUST contain a `handle` numeric identifier. This is then used when responses are received from the device for matching the responses to the messages sent by the controller. The `handle` has no programmatic significance for the device.
```
Comment (jonathan-r-thorpe) [inline docs/Protocol messaging.md]: There are a couple of `Note` indicators in this section that seems needless, as these are already MUST normative clauses.

Could we loose the `Note` indicators?

**Review by jonathan-r-thorpe — APPROVED** [review]
LGTM

**Comment by cristian-recoseanu** [comment]
Reviewed on the call on the 25th of July 2023.

**Pull request merged**